packages/paste-website/src/components/site-sticky-header/index.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+import {Box} from '@twilio-paste/box';
+import {Anchor} from '@twilio-paste/anchor';
+import {useTheme} from '@twilio-paste/theme';
+import {ThemeSwitcher} from '../theme-switcher';
+import GithubIcon from '../svg/github-icon';
+
+interface StyledFlexProps {
+  justifyContent?: string;
+  alignItems?: string;
+}
+export const StyledFlex = styled.div<StyledFlexProps>(({justifyContent, alignItems = 'center'}) => ({
+  display: 'flex',
+  justifyContent,
+  alignItems,
+}));
+
+export const SiteStickyHeader: React.FC<{}> = () => {
+  const theme = useTheme();
+  return (
+    <Box
+      as="aside"
```

Comment:
aside or header or nav? 🤔

Reply:
I went with aside as it is supplimentary text to the side. I see it as a tradition side layout but on it's side. It's currently 

![](https://cl.ly/12bbb5e73a5a/Image%202019-08-15%20at%2018.06.49.png)

Which makes a lot of sense to me

Reply:
Thanks for explaining.

Reply:
How many times can Simon miss-spell site as side in a GitHub comment 😆